### ISSUE TITLE
Feature Request: Demo Deployment

### ISSUE THREAD

**Issue by pReya** [sveltia/sveltia-cms#1]
Hey @kyoshino,

this looks like a wonderful project! Thank you for all this work!

To evaluate the tool it would be very helpful if there was a public demo instance/deployment (e.g. on GitHub Pages). It doesn't need to be full functional, but it would be awesome if we could look at the UI/see the Admin interface in an easy way, without cloning it locally.

Best of luck with this project. I will be following very closely and look forward to having a GitLab backend in the future!

**Comment by kyoshino** [comment]
Hey @pReya, thanks for checking out my project! You are right, I need to set up a demo site so people can give it a try easily. I’ll work on it hopefully later this month. 🤞

**Comment by almereyda** [comment]
Hi @pReya I have filed the part about a feature request for GitLab support as #22, because we would like to see it, too. Feel free to 👍🏽 it.

**Comment by divan** [comment]
Is there any screenshot or video or demo site to even see how Svetlia CMS looks like? I'm torturing search engines and LLMs for 20 minutes straight and still can't get an idea how different Svetlia CMS UI from Decap CMS (which has a demo, so it's immediately clear what's the look & fell)) or others.

**Comment by kyoshino** [comment]
Our [README](https://github.com/sveltia/sveltia-cms) includes a couple of screenshots. The UI is very different from Decap.

In order to keep people’s expectations low, we have decided not to create a demo site before the 1.0 release candidate is ready. If you already have a Decap CMS instance, which Sveltia CMS expects, you can easily try it out locally by using the [Test backend](https://decapcms.org/docs/test-backend/) (with a virtual file system) or [local repository workflow](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#working-with-a-local-git-repository) (with a real file system).

**Comment by MentalGear** [comment]
I agree, this really needs a demo (w/ newsletter sub if interested) website otherwise all the traffic you generate before the 1.0 release evaporates.

**Comment by kyoshino** [comment]
It’s fine. We don’t want to increase our presence at this time.
